ArcGISRuntimeSDKQt_QMLSamples/Layers/BrowseOGCAPIFeatureService/BrowseOGCAPIFeatureService.qml

```diff
@@ -124,7 +127,15 @@ Rectangle {
         // Connect loaded signal to checkForLoadingErrors() function
         root.featureService.loadStatusChanged.connect(checkForLoadingErrors);
 
+        // Once loaded, return the loading property to false
+        root.featureService.loadStatusChanged.connect(function() {
+            if (root.featureService.loadStatus === Enums.LoadStatusLoaded)
+                loading = false;
```

Comment:
there is also `Enums.LoadStatusFailedToLoad` which you should handle as a terminal state

Reply:
There is a function called `checkForLoadingErrors` which handles the `Enums.LoadStatusFailedToLoad`:
```
function checkForLoadingErrors() {
    if (root.featureService.loadStatus === Enums.LoadStatusFailedToLoad) {
        handleError(root.featureService.loadError);
    }
}
```
Will this be sufficient?